extensions/websockets/server/deployment/src/main/java/io/quarkus/websockets/deployment/ServerWebSocketProcessor.java

```diff
@@ -32,6 +38,17 @@ public class ServerWebSocketProcessor {
 
     private static final DotName SERVER_ENDPOINT = DotName.createSimple(ServerEndpoint.class.getName());
 
+    public static final Collection<DotName> CODECS = Arrays.asList(
+            Decoder.TextStream.class,
+            Decoder.Text.class,
+            Decoder.BinaryStream.class,
+            Decoder.Binary.class,
+            Encoder.TextStream.class,
+            Encoder.Text.class,
+            Encoder.BinaryStream.class,
+            Encoder.Binary.class).stream().map(Class::getName).map(DotName::createSimple)
+            .collect(Collectors.toList());
```

Comment:
This would be simpler with `List.of` would it not?

Reply:
Done !